src/ProtocolGateway.Core/IotHub/Routing/IIotHubMessageRouter.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.ProtocolGateway.IotHub.Routing
```

Comment:
in its current form it seems tied to MQTT so makes sense to keep it under Mqtt namespace.